azure-client-runtime/src/main/java/com/microsoft/azure/v2/annotations/AzureHost.java

```diff
@@ -0,0 +1,66 @@
+/**
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See License.txt in the project root for
+ * license information.
+ */
+
+package com.microsoft.azure.v2.annotations;
+
+import com.microsoft.azure.AzureEnvironment;
+import com.microsoft.azure.AzureEnvironment.Endpoint;
+
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+import static java.lang.annotation.ElementType.METHOD;
+import static java.lang.annotation.ElementType.TYPE;
+
+@Target(value={METHOD,TYPE})    // The context in which annotation is applicable i.e. this annotation (Host) can be applied to method, method parameter and interface.
```

Comment:
Because this is not supported by Swagger 2.0 - to hit a completely different endpoint (parametrized ones are allowed) you need a separate swagger doc anyways